packages/manager/modules/vps/src/upscale/model-price/model-price.component.js

```diff
@@ -0,0 +1,24 @@
+import controller from './model-price.controller';
+import template from './model-price.html';
+
+/**
+ * Examples:
+ * <vps-upgrade-model-price price="{ value: 32.30, unit: 'microcents', currency: 'EUR' }" language="'fr_FR'"></vps-upgrade-model-price>
+ * <vps-upgrade-model-price price="{ value: 32.30, unit: 'cents', currency: 'USD' }" language="'fr_CA'"></vps-upgrade-model-price>
+ * <vps-upgrade-model-price price="{ value: 32.30, unit: 'cents', currency: 'EUR' }" language="'en_US'"></vps-upgrade-model-price>
+ */
+export default {
```

Comment:
I see both model-price and configuration-price are identical (component, controller and template). Is it possible to reuse the same component?

Reply:
+1 Yes, only the translate key used in the template differs

Reply:
I do see what you mean but they are semantically different and could evolve in different directions, this would be a bad abstraction to refactor only for the sake of being DRY 🙂 

I invite you to read this article and to tell me what you think about this: https://overreacted.io/goodbye-clean-code/ 🙂